client/blocks/app-banner/style.scss

```diff
@@ -1,15 +1,34 @@
+.app-banner-overlay {
+	position: fixed;
+	height: 100vh;
+	width: 100%;
+	top: 0;
+	left: 0;
+	background: rgba( var( --studio-black-rgb ), 0.6 );
+
```

Comment:
Not sure if it super necessery but it might be good to also add 
`touch-action: none;` 
here so that the user is not able to do other things like zoom and pinch and swipe in here. 

🤔

Reply:
Good idea. Added it: d3ce8a36b8abdd6be53b7db6d172de35d0c565de